pyrightconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+    "disableBytesTypePromotions": true
```

Comment:
without this setting, pyright turns `bytes` into `bytes | bytearray | memoryview`, which blows up our code because only `bytes` has a `decode` method.

The return type of cryptography for x509 name values is `str | bytes`, never was there an intent to support bytearray or memoryview